lib/process-services-cloud/src/lib/rich-text-editor/editorjs-config.ts

```diff
@@ -1,102 +0,0 @@
-/*!
- * @license
- * Copyright © 2005-2025 Hyland Software, Inc. and its affiliates. All rights reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-/** Plugin import */
-import CodeTool from '@editorjs/code';
-import Header from '@editorjs/header';
-import Paragraph from '@editorjs/paragraph';
-import InlineCode from '@editorjs/inline-code';
-import List from '@editorjs/list';
-import Marker from '@editorjs/marker';
-import Underline from '@editorjs/underline';
-import ChangeFontSize from '@valano/change-font-size';
-import ColorPlugin from 'editorjs-text-color-plugin';
```

Comment:
That one dependency and editorjs-text-alignment-blocktune should be removed similarly like above